recipe/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
     - ujson >=1.35
     - dill >=0.2,<0.3
     - requests >=2.13.0,<3.0.0
-    - regex ==2017.4.5
+    - regex >=2017.04.05
```

Comment:
Somehow I lost the previous comment. Sorry about that. To keep it close to the original release, please change `>=` to `==`. Afterwards @honnibal can tell whether or not is safe to use `>=`.